types/voting.ts

```diff
@@ -205,3 +197,35 @@ export type DecodedAdminTransactionsByIdentifierT = Record<
 export type VoteDecodedAdminTransactionsT = {
   decodedAdminTransactions: DecodedAdminTransactionsT | undefined;
 };
+
+export type SupportedChainIds = 1 | 5 | 10 | 137 | 42161;
```

Comment:
I got these chain ids from the `NODE_URLS` env var. @chrismaree does this look right or are we missing some?

Reply:
5 is only relevent on testnets (goerli). we also need:
- 288 (boba)
- 416 (SX Network)
- 43114 (Avalance C-Chain)
- 100 (Gnosis chain).